tockloader/src/board_settings.rs

```diff
@@ -1,3 +1,5 @@
+#[allow(dead_code)]
```

Comment:
This will be fixed on Ami's branch, I would mobe it to the `board` field rather than the whole struct